src/Symfony/Component/DependencyInjection/Tests/Compiler/IntegrationTest.php

```diff
@@ -458,22 +458,6 @@ public function testFactoryWithAutoconfiguredArgument()
         self::assertSame($container->get(FooTagClass::class), $locator->get('my_service'));
     }
 
-    /**
-     * @requires PHP 8
-     */
-    public function testMultipleArgumentBindings()
```

Comment:
I don't think that the behavior that this test case checks is desired. Better deal with this is a fifo way. That's less code for us, to deal with a situation that we don't have to deal with in the first place.

Reply:
The intention here was to make a developer's mistake visible. But we can remove the check if you think we don't need it.